networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -45,6 +50,11 @@ remote error: tls: protocol version not supported
 +
 Now, the AWS Load Balancer Controller accepts TLS 1.2 as the minimum TLS version, resolving this issue. (link:https://issues.redhat.com/browse/OCPBUGS-14846[*OCPBUGS-14846*])
 
+[id="aws-load-balancer-operator-release-notes-1.0.1"]
+== AWS Load Balancer Operator 1.0.1
+
+The following advisory is available for the AWS Load Balancer Operator 1.0.1 on OperatorHub. link:https://access.redhat.com/errata/RHEA-2024:0556[Release of AWS Load Balancer Operator 1.0.1 on OperatorHub]
```

Comment:
Same here, the link to the erratum should be a bullet point. :slightly_smiling_face: 

For consistency, we should also include the word "version" before the version number. There's also no need to mention OperatorHub in the text before the link. Example:

```adoc
The following advisory is available for the AWS Load Balancer Operator version 1.0.1:

* link:https://access.redhat.com/errata/RHEA-2024:0556[Release of AWS Load Balancer Operator 1.0.1 on OperatorHub]
```